Product OS docs refresh (#7800)
PostHog/posthog.com: contents/docs/data/cohorts.mdx

- Users who signed up recently.
- Users who viewed the signup page, but didn't convert.

> You list of cohorts are accessible from the People page 

Reviewer: ```suggestion
You list of cohorts are accessible from the [People](https://us.posthog.com/persons) page.
```

## Where can you use cohorts?

You can use cohorts in many different ways, such as:
- In [trends](/docs/product-analytics/trends), funnels, retention, user paths, stickiness, and lifecycle insights
- As a filter on any product analytics dashboard
- To target feature flags, experiments, and user surveys
- To filter the session replay list and create playlists
- Filter live events on the Activity page
- Filter users on the People page

You can use cohorts to answer questions like:

- "Are people at company X using our app frequently?"
- "What percentage of the users who signed up last week are still using our app?"
- "Which features are power users using?"

## How are cohorts different from groups?

Cohorts are sometimes confused with [groups](/docs/product-analytics/group-analytics), but they each serve different purposes:

> - **Cohorts** represent a specific set of users – e.g., a list of users that all belong to the same company.

Reviewer: I'm not sure if this makes this more or less confusing.
```suggestion
- **Cohorts** represent a specific set of users – e.g., a list of users who's email contains a certain string (like a company's domain).
```

> - **Groups* aggregate events based on entities, such as organizations or companies, but do not necessarily connect to a user. They enable you to analyze trends, insights, and dashboards at an entity-level, as opposed to a user-level.

Reviewer: ```suggestion
- **Groups** aggregate events based on entities, such as organizations or companies, but do not necessarily connect to a user. They enable you to analyze trends, insights, and dashboards at an entity-level, as opposed to a user-level.
```

You can create cohorts using data from insights such as trends, [funnels](/docs/product-analytics/funnels), [user paths](/docs/product-analytics/paths) and [lifecycles](/docs/product-analytics/lifecycle).

> 1. Go to [insights](https://app.posthog.com/insights) and open any insight that's aggregated by users – i.e. `unique users`, `weekly active user`, etc. Click a data point on the visualization to view the [persons](/docs/data/persons) represented in the underlying data.

Reviewer: ```suggestion
1. Go to [insights](https://us.posthog.com/insights) and open any insight that's aggregated by users – i.e. `unique users`, `weekly active user`, etc. Click a data point on the visualization to view the [persons](/docs/data/persons) represented in the underlying data.
```

2. In the modal popup, click 'Save as cohort' at the bottom.

The video below shows how to create a new cohort from a trend, although you can similarly create a cohort from a funnel, user path, or any other insight.

![How to create a cohort from a trend](../../images/features/cohorts/create-cohort-from-trend.mp4)

### Method 2: Create a new cohort from the cohorts page

> 1. Navigate to the [Persons](https://app.posthog.com/persons) page within the sidebar, then click on the [Cohorts tab](https://app.posthog.com/cohorts):

Reviewer: ```suggestion
1. Navigate to the [Persons](https://us.posthog.com/persons) page within the sidebar, then click on the [Cohorts tab](https://us.posthog.com/cohorts).
```

2. Click '+ New Cohort' on the top right.

![left hand navigation for cohorts](../../images/features/cohorts/cohorts-page.png)

## Static and dynamic cohorts

You have two options when creating a cohort:

1. **Static cohorts**, lists of users that don't change over time (e.g. all the users who signed up to your app in January 2024) or you update manually.

2. **Dynamic cohorts**, which are automatically updated based on a specific condition – e.g. all users who have performed a specific event or action in your app.

### Static cohorts

> Static cohorts are created from insights, or by uploading a CSV of users. 

Reviewer: ```suggestion
Static cohorts are created from insights, by uploading a CSV of users, or by duplicating a dynamic cohort.
```

### Dynamic cohorts

Dynamic cohorts can be based on existing cohorts, and support the following criteria:

#### Behavioral

| **Event or action**                                             | **Example**                                                                                                        |
|------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------|
| **Completed event or action**                        | Completed `user signed up` in the last 30 days                                                                     |
| **Did not complete event or action**                 | Did not complete `user signed up` in the last 2 months                                                             |
| **Completed event or action multiple times**         | Completed `watched video` 5 times in the last 2 weeks                                                              |
| **Completed a sequence of events or actions**        | Completed `user signed up` in the last 5 days followed by `watched video` within 1 day of the initial event        |

> | **Did not complete a sequecne of events or actions** | Did not complete `user signed up` in the last 5 days followed by `watched video` within 1 day of the initial event |

Reviewer: ```suggestion
| **Did not complete a sequence of events or actions** | Did not complete `user signed up` in the last 5 days followed by `watched video` within 1 day of the initial event |
```

You can select `days`, `weeks`, `months` or `years` time ranges here.

> 

Reviewer: I think this is an important note, could be fixed a little.
```suggestion
> **Can use a dynamic behavioral cohort as a feature flag target?** No. To use a cohort based on their behaviors as a target for a feature flag, A/B test, or survey, you must duplicate the dynamic cohort to a static one and use it.

```

#### Person properties

Has or doesn't have a person property with the following operators:

| **Operator**                                 | **Example**                                           |
|----------------------------------------------|-------------------------------------------------------|
| `equals` or `does not equal`                 | `Pineapple on pizza` property `does not equal` "true" |
| `contains` or `does not contain`             | `email` property `does not contain` "gmail"           |
| `matches regex` or `does not match regex`    | `Country code` property `matches regex` "us|uk|aus"   |
| `greater than` or `greater than or equal to` | `Age` property is `greater than` "21"                 |
| `less than` or `less than or equal to`       | `Age property is `less than or equal to` "21"         |
| `is set` or `is not set`                     | `UTM Source` property `is not set`                    |
| `between` or `not between`                   | `Age` property is between "18" and "30"               |
| `minimum` or `maximum` value                 | `Organization size` property `minimum` of "5"         |

#### Lifecycle

> | ****Criteria****                       | **Example**                                                                                         |

Reviewer: ```suggestion
| **Criteria**                       | **Example**                                                                                         |
```
